seagence/manifest.json

```
@@ -0,0 +1,62 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "94f4e504-c98c-466f-b934-5e5ee0331944",
+  "app_id": "seagence",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
```

Ah, sorry we also need this line here so the uninstallation instructions show up in the tile.
```suggestion
    "configuration": "README.md#Setup",
    "uninstallation": "README.md#Uninstallation",
```